src/accessibility/describe.js

```diff
@@ -0,0 +1,304 @@
+ * </div>
+ */
+
+p5.prototype.describeElement = function(n, t, d) {
+  p5._validateParameters('describeElement', arguments);
+  if (typeof t !== 'string' || typeof n !== 'string') {
+    return;
+  }
+  const cnvId = this.canvas.id;
+  t = this._descriptionText(t);
+  let nm = this._elementName(n);
+  //Creates a sub DOM inside of the canvas with a table, populates
+  //a row header cell with the name of the elements and adds the description
+  //of the element in adjecent cell.
+  if (document.getElementById(cnvId + ds) === null) {
+    document.getElementById(cnvId).innerHTML =
+      '<div id="' +
+      cnvId +
+      ds +
+      '" role="region" aria-label="Canvas Description"><table id="' +
+      cnvId +
+      ft +
+      '"><caption>Canvas elements and their descriptions</caption></table></div>';
+  } else if (document.getElementById(cnvId + ft) === null) {
+    document
+      .getElementById(cnvId + fds)
+      .insertAdjacentHTML(
+        'afterend',
+        '<table id="' +
+          cnvId +
+          ft +
+          '"><caption>Canvas elements and their descriptions</caption></table>'
+      );
+  }
+  if (document.getElementById(cnvId + fte + n) === null) {
+    let tr = document.createElement('tr');
+    tr.id = cnvId + fte + n;
+    document.getElementById(cnvId + ft).appendChild(tr);
+  }
+  if (
+    document.getElementById(cnvId + fte + n).innerHTML !==
+    '<th scope="row">' + nm + '</th><td>' + t + '</td>'
+  ) {
+    document.getElementById(cnvId + fte + n).innerHTML =
+      '<th scope="row">' + nm + '</th><td>' + t + '</td>';
+  }
+  //If display is LABEL creates a div adjacent to the canvas element with
+  //a table, a row header cell with the name of the elements,
+  //and adds the description of the element in adjecent cell.
+  if (d === this.LABEL) {
+    if (document.getElementById(cnvId + lb) === null) {
+      document
+        .getElementById(cnvId)
+        .insertAdjacentHTML(
+          'afterend',
+          '<div id="' +
+            cnvId +
+            lb +
+            '" class="p5Label"><table id="' +
+            cnvId +
+            lt +
+            '"></table></div>'
+        );
+    } else if (document.getElementById(cnvId + lt) === null) {
+      document
+        .getElementById(cnvId + lds)
+        .insertAdjacentHTML(
+          'afterend',
+          '<table id="' + cnvId + lt + '"></table>'
+        );
+    }
+    if (document.getElementById(cnvId + lte + n) === null) {
+      let tr = document.createElement('tr');
+      tr.id = cnvId + lte + n;
+      document.getElementById(cnvId + lt).appendChild(tr);
+    }
+
+    if (
+      document.getElementById(cnvId + lte + n).innerHTML !==
+      '<th scope="row">' + nm + '</th><td>' + t + '</td>'
+    ) {
+      document.getElementById(cnvId + lte + n).innerHTML =
+        '<th scope="row">' + nm + '</th><td>' + t + '</td>';
+    }
+  }
+};
+/**
+ * Helper function for describeElement().
+ */
+p5.prototype._elementName = function(n) {
```

Comment:
if this is only used internally, it doesn't need to be attached to `p5.prototype`